src/chunk.c

```diff
@@ -2266,6 +2266,15 @@ ts_chunk_get_by_relid(Oid relid, bool fail_if_not_found)
 
 	schema = get_namespace_name(get_rel_namespace(relid));
 	table = get_rel_name(relid);
+
+	if (schema == NULL || table == NULL)
+	{
+		if (fail_if_not_found)
+			ereport(ERROR, (errcode(ERRCODE_UNDEFINED_OBJECT), errmsg("invalid Oid")));
```

Comment:
Using same error message as PostgreSQL is probably helpful if we ever run into this error.

```suggestion
			ereport(ERROR, (errcode(ERRCODE_UNDEFINED_OBJECT), errmsg("relation with OID %u does not exist")));
```

Reply:
Changing it to `chunk not found` because that's what is returned by the final `chunk_get_by_name` function as well. A test case was failing because it didn't find this `chunk not found` expected error.

Reply:
Shouldn't you print the chunk relid in that case? It makes it easier to debug if that message is printed.

Reply:
You can also change the other error message, if they do not match.

Reply:
A side effect of adding oids in the error output is that our test cases start failing because the OID can change from setup to setup.

Reply:
Good point. It could be fixed by "cleaning" the test output (tricky) or put the OID in the detailed message and not print the detailed message in the test output.

Reply:
Adding the OID to the detailed message seems like a good compromise. Will do.